addons/website_sale/views/product_views.xml

```diff
@@ -170,20 +170,34 @@
                 <field name="description_ecommerce" invisible="1" />         <!-- Adding this field as we needed in product_barcodelookup module to store value in this field -->
             </group>
             <xpath expr="//page[@name='sales']/group[@name='sale']" position="inside">
-                <group string="eCommerce Shop" name="shop" invisible="not sale_ok">
-                    <field name="website_url" invisible="1"/>
-                    <field name="website_id"
-                        options="{'no_create': True}"
-                        groups="website.group_multi_website"
-                        placeholder="All"/>
-                    <field name="website_sequence" groups="base.group_no_one"/>
-                    <field name="public_categ_ids" widget="many2many_tags" string="Categories"/>
-                    <field name="website_ribbon_id" groups="base.group_no_one" options="{'no_quick_create': True}"/>
-                </group>
                 <group colspan="2" name="product_template_images" string="eCommerce Media" invisible="not sale_ok">
                     <field colspan="2" name="product_template_image_ids" class="o_website_sale_image_list" context="{'default_name': name}" mode="kanban" add-label="Add Media" nolabel="1" widget="x2_many_media_viewer"/>
                 </group>
             </xpath>
+            <xpath expr="//page[@name='sales']//group[@name='description']" position="after">
+                <group string="Ecommerce Description">
+                    <field colspan="2" name="description_ecommerce" nolabel="1"
+                        placeholder="A detailed,formatted description to promote your product on this page.
+                        Use '/' to discover more features."/>
+                </group>
+            </xpath>
+            <group name="tags" position="inside">
+                <field name="is_published"/>
+                <field name="website_url" invisible="1"/>
```

Comment:
Do need this invisible field here? :thinking:

Reply:
No
Invisible fields are not needed.